pkg/plugin/builtin/v1/source.go

```diff
@@ -96,7 +96,7 @@ func (s *sourcePluginAdapter) Start(ctx context.Context, p record.Position) erro
 	s.stream = newSourceRunStream(ctx)
 	go func() {
 		s.logger.Trace(ctx).Msg("calling Run")
-		err := s.impl.Run(s.withLogger(ctx), s.stream)
+		err := runSandboxNoResp(s.impl.Run, s.withLogger(ctx), cpluginv1.SourceRunStream(s.stream))
```

Comment:
Do we need to sandbox the `SourceRunStream` too, like the comment for `DestinationRunStream` notes?

Reply:
Which comment do you mean? I don't think we need to sandbox anything else, actually, I don't think we _can_ sandbox anything else. Conduit calls 5 functions in the plugin and those functions are all sandboxed now.

Reply:
I thought there was a TODO comment about sandboxing the SourceRunStream, but I don't see it anymore so disregard 👍